glue_qt/viewers/common/data_slice_widget.py

```diff
@@ -160,7 +160,7 @@ def _adjust_play(self, action):
             self._play_timer.stop()
         else:
             self._play_timer.start()
-            self._play_timer.setInterval(500 / abs(self._play_speed))
+            self._play_timer.setInterval(round(500 / abs(self._play_speed))
```

Comment:
```suggestion
            self._play_timer.setInterval(round(500 / abs(self._play_speed)))
```
Thanks! When you fix this, could you rebase to get the CI fixes in?

Reply:
Thanks! should be fixed now and rebased.